### PR TITLE
Update PROVISIONING.md

### PR DESCRIPTION
**Pull request by glimchb**
* material and questions surrounding UEFI HTTPS boot.
* regroup vmedia under a manual method
* identified/proposed two categories of provisioning: private network and multi-tenant public network. Probably more important to focus on latter.

Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

File: PROVISIONING.md [hunk 1]
```diff
@@ -23,10 +25,10 @@ tbd... re-celling, faults...
 NVIDIA has a manual provisioning process based on a virtual *-over-PCIe device set, called [RSHIM](https://github.com/Mellanox/rshim). RSHIM creates, among other things, a virtual point-to-point ethernet device, and a virtual console device, between host and DPU. See also [usage](https://docs.nvidia.com/networking/display/BlueFieldDPUOSLatest/Deploying+DPU+OS+Using+BFB+from+Host) of RSHIM.
 Many customers are using this process to deploy their own OS image and initialize system configuration, since they trust the OS running on the x86 host.
 
-## USB/Virtual media Provisioning
+Andy: Do we need to cover manual methods in our scope? Many, many methods.
```
Comment (mestery) [inline PROVISIONING.md:28]: This would seem to open up an endless variety of possibilities as you imply.

File: PROVISIONING.md [hunk 2]
```diff
@@ -23,10 +25,10 @@ tbd... re-celling, faults...
 NVIDIA has a manual provisioning process based on a virtual *-over-PCIe device set, called [RSHIM](https://github.com/Mellanox/rshim). RSHIM creates, among other things, a virtual point-to-point ethernet device, and a virtual console device, between host and DPU. See also [usage](https://docs.nvidia.com/networking/display/BlueFieldDPUOSLatest/Deploying+DPU+OS+Using+BFB+from+Host) of RSHIM.
 Many customers are using this process to deploy their own OS image and initialize system configuration, since they trust the OS running on the x86 host.
 
-## USB/Virtual media Provisioning
+Andy: Do we need to cover manual methods in our scope? Many, many methods. 
 
+### USB/Virtual media Provisioning (this is typically a manual method, enabled by a one-to-one interaction with a BMC)
```
Comment (mestery) [inline PROVISIONING.md:30]: Minor nit: Maybe move the wording in parenthesis into a sentence outside of the header.

**Review by sandersms — APPROVED** [review]
Looks good

File: PROVISIONING.md [hunk 3]
```diff
@@ -71,6 +73,11 @@ Use case: large scale deployments (where automation and security are major drive
 - Device reboots into newly installed software
   - Question: if the version is the same, can the entire process skip ? where this happens?
 
+Do we want to favor UEFI methods (like HTTPS boot) over others that require a client running in an OS, or a BMC (like sZTP)?
```
Comment (mestery) [inline PROVISIONING.md:76]: I feel as if when we start to produce code for this, we would definitely want to prioritize the specific methods of booting. So I think it's less favoring and more prioritizing.

**Review by mestery — APPROVED** [review]
LGTM

File: PROVISIONING.md (hunk 1 above)
Comment (glimchb) [reply PROVISIONING.md:28]: just wanted to list a couple for completeness

**Pull request merged**